config.php

```diff
@@ -120,12 +120,16 @@
 
 // DEBUGGING & DEVELOPERS
 $this->showStats = false;
-$this->debug = false;
+$this->debug = true; //false;
```

Comment:
Not so sure whether committing the changed value is a good idea at all. It may (and eventually will) lead to erroneous merging this to stable branches.